backend/tests/Tests/Serialization.TestValues.fs

```diff
@@ -231,7 +231,12 @@ module ProgramTypes =
     let e = PT.EInt(34545UL, 5)
     PT.ELet(
       14219007199254740992UL,
-      PT.LPVariable(189271UL, "x1"),
+      PT.LPTuple(
+        189271UL,
+        PT.LPVariable(18274132UL, "x0"),
```

Comment:
Maybe have a nested LPTuples here too?

Reply:
add a nested tuple, and also a tuple in the rest part